server/routerlicious/packages/services-shared/src/redisSocketIoAdapter.ts

```diff
@@ -423,6 +423,9 @@ export class RedisSocketIoAdapter extends Adapter {
      * It will publish a message over the pub connection and wait until it receives it
      */
     private async runRoomHealthCheck(room: string) {
+        if (!RedisSocketIoAdapter.options.healthChecks) {
```

Comment:
I might be missing something but how does this change relate to the rest of the PR? Is this just a bonus change?

Reply:
Makes sense! Thanks for providing the context!